java/client/src/org/openqa/selenium/edge/edgehtml/EdgeHTMLDriverInfo.java

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.openqa.selenium.edge.edgehtml;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.edge.EdgeDriverInfo;
+
+public class EdgeHTMLDriverInfo extends EdgeDriverInfo {
```

Comment:
We were looking for a way to distinguish between the old Edge and the new one. Since we call the new one base on its engine (Chromium), so I thought we'd do the same for the old Edge with EdgeHTML engine.

Reply:
Right, but `EdgeHtmlDriverInfo` follows the style of the project more closely than `EdgeHTMLDriverInfo`.